webapi/src/main/scala/org/knora/webapi/messages/admin/responder/permissionsmessages/PermissionsMessagesADM.scala

```diff
@@ -44,7 +44,7 @@ case class CreateAdministrativePermissionAPIRequestADM(
   implicit protected val sf: StringFormatter = StringFormatter.getInstanceForConstantOntologies
 
   id match {
-    case Some(iri) => sf.validatePermissionIRI(iri)
+    case Some(iri) => sf.validatePermissionIri(iri).fold(e => throw BadRequestException(e), v => v)
```

Comment:
would it be out of scope to remove these throws?

Reply:
A bit yes. All messages contain loads of throws anyway, because were treated like the old "validation" place, so I just went for the simplest way. There are not ZIOfied yet, so this is step which rather makes getting rid of throws here cumbersome. How do you feel?